switchcraft/data/computercraft/lua/rom/programs/http/pastebin.lua

```diff
@@ -23,9 +23,11 @@ local PASTE_ID_PATTERNS = {
 local function printUsage()
   local programName = arg[0] or fs.getName(shell.getRunningProgram())
   print("Usages:")
-  print(programName .. " put <filename>")
-  print(programName .. " get <code> <filename>")
+  print(programName .. " put <filename> [password] [visibility]")
+  print(programName .. " get <code> <filename> [password]")
   print(programName .. " run <code> <arguments>")
+  print()
+  print("Due to technical limitations, "..programName.." run can't be used with protected pastes")
```

Comment:
I would add spaces for readability.
```suggestion
  print("Due to technical limitations, " .. programName .. " run can't be used with protected pastes")
```

Reply:
second concatenation here missing spaces?

Reply:
Updated my suggestion